dts/arm/st/f1/stm32f100Xe.dtsi

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2020 Jonas Eriksson, Up to Code AB
+ *
+ * SoC device tree include for STM32F100xB SoCs
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+#include <st/f1/stm32f100b.dtsi>
+
+/ {
+	sram0: memory@20000000 {
+		reg = <0x20000000 DT_SIZE_K(32)>;
+	};
+
+	soc {
+		flash-controller@40022000 {
+			flash0: flash@8000000 {
+				reg = <0x08000000 DT_SIZE_K(512)>;
+				erase-block-size = <DT_SIZE_K(1)>;
```

Comment:
`erase-block-size` doesn't change from stm32f100Xe.dtsi, remove this line

Reply:
Oh, I thought the merging stopped earlier. But that makes sense that this also gets merged.